libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -69,6 +69,14 @@ const AP_Param::GroupInfo HarmonicNotchFilterParams::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("REF", 6, HarmonicNotchFilterParams, _reference, 0),
 
+    // @Param: TRACK_MODE
+    // @DisplayName: Harmonic Notch Filter dynamic frequency tracking mode
+    // @Description: Harmonic Notch Filter dynamic frequency tracking mode. Dynamic updates can be throttle or ESC telemetry based.
```

Comment:
@andyp1per please make this clear that this is not used for Heli and is only used for multicopters.  Otherwise we need all available options (heli and multi) and then each option is annotated with the one it applies to or both.   Will heli's be able to use the BLHeli option?  I would assume it would require scaling if it is passing motor RPM?

Reply:
Hi @bnsgeyer - I'll use @lthall 's suggestion and then its configurable by all